portfolio/src/main/webapp/script.js

```diff
@@ -48,4 +49,24 @@ function createListElement(text) {
   return liElement;
 }
 
-  
+function getLogin() {
+    //gets login status
+    fetch('/login').then(response => response.json()).then((jsonFile) => {
+    const divTag = document.getElementById('loginForm');
+    divTag.innerHTML = '';
+    if (jsonFile.login == false)
```

Comment:
If !jsonFile.login works too

Also it might be easier to write the HTML in your HTML file and hide it with CSS then you just have to show/hide that here rather than programmatically build DOM elements.